---
title: SanitizationException
second_title: Aspose.PDF für .NET-API-Referenz
description: Initialisiert eine neue Instanz vonSanitizationExceptionaspose.pdf.sanitization/sanitizationexception Klasse.
type: docs
weight: 10
url: /de/net/aspose.pdf.sanitization/sanitizationexception/sanitizationexception/
---
## SanitizationException() {#constructor}

Initialisiert eine neue Instanz von[`SanitizationException`](../../sanitizationexception) Klasse.

```csharp
public SanitizationException()
```

### Siehe auch

* class [SanitizationException](../../sanitizationexception)
* namensraum [Aspose.Pdf.Sanitization](../../sanitizationexception)
* Montage [Aspose.PDF](../../../)

---

## SanitizationException(string) {#constructor_2}

Initialisiert eine neue Instanz von[`SanitizationException`](../../sanitizationexception) Klasse.

```csharp
public SanitizationException(string message)
```

| Parameter | Typ | Beschreibung |
| --- | --- | --- |
| message | String | Die Nachricht. |

### Siehe auch

* class [SanitizationException](../../sanitizationexception)
* namensraum [Aspose.Pdf.Sanitization](../../sanitizationexception)
* Montage [Aspose.PDF](../../../)

---

## SanitizationException(string, Exception) {#constructor_3}

Initialisiert eine neue Instanz von[`SanitizationException`](../../sanitizationexception) Klasse mit einer angegebenen Fehlermeldung und einem Verweis auf die innere Ausnahme, die die Ursache dieser Ausnahme ist.

```csharp
public SanitizationException(string message, Exception innerException)
```

| Parameter | Typ | Beschreibung |
| --- | --- | --- |
| message | String | Die Fehlermeldung, die den Grund für die Ausnahme erklärt. |
| innerException | Exception | Die Ausnahme, die die Ursache der aktuellen Ausnahme ist, oder ein NULL-Verweis (Nothing in Visual Basic), wenn keine innere Ausnahme angegeben ist. |

### Siehe auch

* class [SanitizationException](../../sanitizationexception)
* namensraum [Aspose.Pdf.Sanitization](../../sanitizationexception)
* Montage [Aspose.PDF](../../../)

---

## SanitizationException(Exception) {#constructor_1}

Initialisiert eine neue Instanz von[`SanitizationException`](../../sanitizationexception) Klasse mit einer angegebenen Fehlermeldung und einem Verweis auf die innere Ausnahme, die die Ursache dieser Ausnahme ist.

```csharp
public SanitizationException(Exception innerException)
```

| Parameter | Typ | Beschreibung |
| --- | --- | --- |
| innerException | Exception | Die Ausnahme, die die Ursache der aktuellen Ausnahme ist, oder ein NULL-Verweis (Nothing in Visual Basic), wenn keine innere Ausnahme angegeben ist. |

### Siehe auch

* class [SanitizationException](../../sanitizationexception)
* namensraum [Aspose.Pdf.Sanitization](../../sanitizationexception)
* Montage [Aspose.PDF](../../../)

<!-- DO NOT EDIT: generated by xmldocmd for Aspose.PDF.dll -->
